test/baseSepoliaDeployments.test.ts

```diff
@@ -0,0 +1,173 @@
+import {
+  HatsModulesClient,
+  solidityToTypescriptType,
+} from "@hatsprotocol/modules-sdk";
+import { createPublicClient, createWalletClient, http } from "viem";
+import { privateKeyToAccount } from "viem/accounts";
+import { base, baseSepolia } from "viem/chains";
+import { createAnvil } from "@viem/anvil";
+import { bundleModules } from "../bundler";
+import type {
+  PublicClient,
+  WalletClient,
+  PrivateKeyAccount,
+  Address,
+} from "viem";
+import type { Anvil } from "@viem/anvil";
+import type { Module, Registry } from "@hatsprotocol/modules-sdk";
+import "dotenv/config";
+
+describe("Base deployments", () => {
```

Comment:
should be "Base Sepolia" 👀

Reply:
nice catch! fixed in [9dcde05](https://github.com/Hats-Protocol/modules-registry/pull/77/commits/9dcde05f793f0ccf69eebe84bcdb2d640cedb2a6)